app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,68 @@
+class TweetController < ApplicationController
+
+  get '/tweets' do
+    if logged_in?
+      @tweets = Tweet.all
+      @users = User.all
```

Comment:
Great :)

> On Sep 6, 2016, at 8:27 AM, ThisIsTyDell <notifications@github.com> wrote:
> 
> In app/controllers/tweet_controller.rb <https://github.com/learn-co-students/sinatra-fwitter-group-project-v-000/pull/201#discussion_r77657923>:
> 
> > @@ -0,0 +1,68 @@
> > +class TweetController < ApplicationController
> > +
> > +  get '/tweets' do
> > +    if logged_in?
> > +      @tweets = Tweet.all
> > +      @users = User.all
> I wanted all of the users available here, so that I can display a list of "Popular Users" on the all tweets page, in addition to displaying a list of tweets. So on the left 1/3 of the browser it shows a list of users. On the right 2/3 of the browser it list all tweets.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub <https://github.com/learn-co-students/sinatra-fwitter-group-project-v-000/pull/201/files/4b544d844c594b28d94e622d3168572440005f40#r77657923>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AOUVeys7tivB3U57NTPQTwz-CaW-M-grks5qnYZNgaJpZM4J1YER>.
>